core-api/src/main/java/com/optimizely/ab/config/audience/AndCondition.java

```diff
@@ -27,7 +27,7 @@
 /**
  * Represents an 'And' conditions condition operation.
  */
-public class AndCondition<T> implements Condition<T> {
+public class  AndCondition<T> implements Condition<T> {
```

Comment:
...

Reply:
This is just a test, don’t worry about this PR please.